src/app/map/page.tsx

```diff
@@ -15,7 +15,7 @@ const MapPage = () => {
         Map Page
       </Text>
       <Link href={`/`}>
-        <Button variant="light">Back Home</Button>
+        <Button buttonType="secondary">Back Home</Button>
```

Comment:
```suggestion
        <Button buttonType=BUTTON_VARIANTS.SECONDARY>Back Home</Button>
```
I think we can use `BUTTON_VARIANTS` instead of hard coding as you export it